packages/block-library/src/image/utils.js

```diff
@@ -56,3 +61,40 @@ export function getUpdatedLinkTargetSettings( value, { rel } ) {
 		rel: updatedRel,
 	};
 }
+
+/**
+ * Determines new href and linkDestination values for an image block from the
+ * supplied link destination.
+ *
+ * @param {Object} image       Image.
+ * @param {string} destination Link destination.
+ * @return {string}            New url attributes to assign to image block.
+ */
+export function getUrl( image, destination ) {
```

Comment:
Maybe I'm not searching properly but this doesn't seem to be used right?

Reply:
yeh, another hangover from the attribute update changes yesterday. Gone now.